docs/source/tutorial.rst

```diff
@@ -1427,8 +1427,25 @@ the only differences being:
                    : longitude(5) = [157.5, ..., -22.5] degrees_east
                    : time(1) = [2019-01-01 00:00:00]
 
+An index that is a list of integers may also be provided as a `numpy`
+array, a `dask` array, a `cf.Data` instance, or a metadata construct
+that has data:
+
+.. code-block:: python
+   :caption: *Create a new field construct for longitudes greater than
+             180.*
```

Comment:
```suggestion
             180 degrees east.*
```

Simply for good practice with always including units, even if it is obvious from context :slightly_smiling_face: